### ISSUE TITLE
A query about the program not working properly

### ISSUE THREAD

**Issue by AbelSean** [201528014227051/brian-stdp-mnist#1]
Thanks to the author for the open source work.
When I run the Diehl&Cook_spiking_MNIST file, it runs for more than four hours and never completes. The specific output is shown below:
![PrtSc](https://user-images.githubusercontent.com/103877169/171602822-8137580c-6305-4313-a9fa-f4951864167c.png)

By the way, compared with the code file provided by the author, there is an extra spiking_MNIST file. What is the function of this file?
I would appreciate it if the problem could be solved.

**Comment by Yunlong-He** [comment]
the program doesn't complete, because there is no any spike found in neurons, I suspect there are some wrong parameters or bugs in the code.

below is the code cause infinit loop:

    if np.sum(current_spike_count) < 5:
        quit()
        input_intensity += 1
        for i,name in enumerate(input_population_names):
            input_groups[name+'e'].rates = np.zeros(n_input,)*Hz
        b.run(resting_time)

**Comment by 201528014227051** [comment]
yes, I noticed this two, the process of the code rewrite can be found in&nbsp;https://wangbinqiang.blog.csdn.net/article/details/122478180
How to solve the problem is a open question now.



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "201528014227051/brian-stdp-mnist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月23日(星期四) 上午10:43
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [201528014227051/brian-stdp-mnist] A query about the program not working properly (Issue #1)





 
the program doesn't complete, because there is no any spike found in neurons, I suspect there are some wrong parameters or bugs in the code.
 
below is the code cause infinit loop:
 if np.sum(current_spike_count) < 5:     quit()     input_intensity += 1     for i,name in enumerate(input_population_names):         input_groups[name+'e'].rates = np.zeros(n_input,)*Hz     b.run(resting_time)

**Comment by krogers-tai** [comment]
Any update on this issue?
